xbmc/addons/AddonManager.cpp

```diff
@@ -815,6 +815,20 @@ void CAddonMgr::FindAddons(ADDON_INFO_LIST& addonmap, const std::string& path)
         AddonInfoPtr addonInfo = CAddonInfoBuilder::Generate(path);
         if (addonInfo)
         {
+          const auto& it = addonmap.find(addonInfo->ID());
+          if (it != addonmap.end())
+          {
+            if (it->second->Version() > addonInfo->Version())
+            {
+
+              CLog::Log(LOGWARNING, "CAddonMgr::{}: Addon '{}' already present on '{}' with higher version {}, other on '{}' with version {} becomes ignored",
+                          __FUNCTION__, addonInfo->ID(), it->second->Path(), it->second->Version().asString(), addonInfo->Path(), addonInfo->Version().asString());
+              continue;
+            }
+            CLog::Log(LOGDEBUG, "CAddonMgr::{}: Addon '{}' already present on '{}' with version {} becomes ignored and replaced with version {} on '{}'",
```

Comment:
I'm not a native English speaker, but for it feels right to just omit the word "becomes" from this string.